src/main/java/uk/gov/digital/ho/hocs/info/security/KeycloakService.java

```diff
@@ -79,6 +80,22 @@ public void removeUserFromTeam(UUID userUUID, UUID teamUUID) {
         }
     }
 
+    public Set<String> getRolesForUser(UUID userUUID) {
+        Set<String> roles  = new HashSet<>();
+        try {
+            RealmResource hocsRealm = keycloakClient.realm(hocsRealmName);
+            List<RoleRepresentation> userRoles = hocsRealm.users().get(userUUID.toString()).roles().realmLevel().listEffective();
```

Comment:
I think x-auth-userid is not the uuid.  I think the above method is wrong too.

Reply:
or perhaps this method is right, but requestData.userIdUUID() is not

Reply:
from the gatekeeper code: req.Header.Set("X-Auth-Userid", user.name)
				req.Header.Set("X-Auth-Username", user.name)
so: 
public static final String USER_ID_HEADER = "X-Auth-UserId";
public UUID userIdUUID() {
        return UUID.fromString(MDC.get(USER_ID_HEADER));
    }
fails to create the uuid